tests/unit/entityInfo.spec.ts

```diff
@@ -6,43 +6,76 @@ import Vuetify from 'vuetify'
 import { getVuexStore } from '@/store'
 
 // Components
-import { mount } from '@vue/test-utils'
+import { createLocalVue, mount } from '@vue/test-utils'
 import { EntityInfo } from '@/components/common'
 
-Vue.use(Vuetify)
+// Other
+import mockRouter from './MockRouter'
+import VueRouter from 'vue-router'
 
+Vue.use(Vuetify)
 const vuetify = new Vuetify({})
-
-describe.skip('Entity Info component with an NR', () => {
-  let wrapper: any
-  let store: any
+document.body.setAttribute('data-app', 'true')
+
+describe('Entity Info component in a Correction as a named company', () => {
+  let wrapper
+  let store
+
+  const mockFiling = {
+    header: {
+      name: 'incorporationApplication',
+      filingId: 12345
+    },
+    business: {
+      identifier: 'BC1234567',
+      legalType: 'BEN'
+    },
+    incorporationApplication: {
+      contactPoint: {
+        email: 'mock@email.com',
+        phone: '123-456-7890'
+      },
+      nameRequest: {
+        legalType: 'BEN',
+        nrNumber: 'NR 1234567',
+        legalName: 'My Mock Name Inc.'
+      },
+      offices: {},
+      parties: [],
+      shareClasses: [],
+      incorporationAgreement: {}
+    }
+  }
 
   beforeEach(() => {
     store = getVuexStore()
-
-    // Entity type will always be set with or without an NR
-    store.state.stateModel.tombstone.entityType = 'BEN'
-    // Temp Id will always be set with or without an NR
-    store.state.stateModel.tombstone.businessId = 'T1234567'
-    store.state.stateModel.nameRequest.nrNumber = 'NR 1234567'
-    store.state.stateModel.nameRequest.details.approvedName = 'Xyz Ltd.'
-
-    wrapper = mount(EntityInfo, { vuetify, store })
+    let state = store.state.stateModel
+
+    state.tombstone.keycloakRoles = ['staff']
+    state.businessInformation = mockFiling.business
+    state.tombstone.businessId = mockFiling.business.identifier
+    state.defineCompanyStep.businessContact = mockFiling.incorporationApplication.contactPoint
+    state.originalIA.incorporationApplication.nameRequest = mockFiling.incorporationApplication.nameRequest
+
+    const localVue = createLocalVue()
+    localVue.use(VueRouter)
+    const router = mockRouter.mock()
+    router.push({ name: 'correction' })
+
+    wrapper = mount(EntityInfo, {
+      localVue,
+      vuetify,
+      store,
+      router
+    })
   })
 
-  afterEach(() => {
-    wrapper.destroy()
+  afterEach(async () => {
+    await wrapper.destroy()
```

Comment:
My hover help (from Vetur extension I think) informs me that "await has no effect on the type of this expression". In other words, "destroy" is not async (does not return a promise).

If you need to wait for things to stability here, use `await Vue.nextTick()`, otherwise please remove this code.

Reply:
done and done, updated in next commit.